cli/src/app.rs

```diff
@@ -347,7 +347,49 @@ pub fn add_subcommands(command: Command) -> Command {
                             .help("Name for the new group")
                             .required(true),
                     ),
-                ),
+                )
+                .subcommand(
+                    Command::new("members").about("List group members").args(&[
+                        Arg::new("group")
+                            .short('g')
+                            .long("group")
+                            .value_name("GROUP")
+                            .help("Group to list the members for")
+                            .required(true),
+                        Arg::new("json")
+                            .action(ArgAction::SetTrue)
+                            .short('j')
+                            .long("json")
+                            .help("Produce group list in json format (default: false)"),
+                        ])
+                    .subcommand(
+                        Command::new("list").about("List all existing projects").args(&[
```

Comment:
Incorrect about message.

```suggestion
                        Command::new("list").about("List group members").args(&[
```